orca-core/src/main/groovy/com/netflix/spinnaker/orca/listeners/ExecutionListener.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.listeners;
+
+import com.netflix.spinnaker.orca.ExecutionStatus;
+import com.netflix.spinnaker.orca.pipeline.model.Execution;
+import org.springframework.core.Ordered;
+
+public interface ExecutionListener extends Ordered {
+  default void beforeExecution(Persister persister, Execution execution) {
```

Comment:
Ya, it could be now.

Originally I was delegating from the old listener interface to the new and passing as a parameter was trivial.

I'll make a note to refactor this.